openfisca_us/parameters/gov/states/il/tax/income/rate.yaml

```
@@ -0,0 +1,29 @@
+description: Illinois Individual Income Tax Rates
+values:
+  2017-07-01:
```

Can do this if you don't have metadata for a data point
```suggestion
  2017-07-01: 0.0495
```